qt/applications/workbench/workbench/app/start.py

```diff
@@ -239,7 +239,11 @@ def start(options: argparse.ArgumentParser):
     if options.single_process:
         initialise_qapp_and_launch_workbench(options)
     else:
-        workbench_process = multiprocessing.Process(target=initialise_qapp_and_launch_workbench, args=[options])
+        # We require the start method to be 'spawn' so that we do not inherit resources such as 'std::once_flag' from
```

Comment:
This is a spot on explanation. 👍🏻 

Can I suggest however that we provide a few more pointers as to where the resources are as things like `std::once_flag` are not in this file or even in the Python code. Maybe a preceding sentence to what you have:

```python
# mantid's FrameworkManagerImpl::Instance Python export uses a process-wide static flag to ensure code
# only runs once (see Instance function in Framework/PythonInterface/mantid/api/src/Exports/FrameworkManager.cpp).
# The default start method on Unix ('fork') inherits resources such as these flags from the parent.
# We require the start method to be 'spawn' so that we do not inherit these resources the parent process,
# this is already the default on Windows/macOS.
# This will mean the relevant 'atexit' code will execute in the child process, and therefore the
# FrameworkManager and UsageService will be shutdown as expected.
```